pages/services/dse/3.0.0-6.7.2/release-notes/index.md

```diff
@@ -0,0 +1,206 @@
+---
+layout: layout.pug
+navigationTitle: Release Notes
+excerpt: Release Notes for DSE v. 3.0.0-6.7.2
+title: Release Notes
+menuWeight: 10
+model: /services/dse/data.yml
+render: mustache
+---
+
+# Version 3.0.0-6.7.2
+
+## Updates
+- Upgraded DSE to version `6.7.2` and OpsCenter to version `6.7.1`
+- Upgraded SDK to `0.55.4` 
+
+## New Features
+- Added pod plan commands: `nodetool-ser` and `nodetool-par` for executing `nodetool` commands either serially or in parallel across all nodes in the cluster [PR #314](https://github.com/mesosphere/dse-private/pull/314):
+	- Now users can issue nodetool commands against all nodes in their cluster via command:
+	```
+	dcos datastax-dse plan start nodetool-ser \
+	  -p NODETOOL_CONNECTION_OPTS='<connection_options>'  \
+	  -p NODETOOL_SUBCOMMAND='<subcommand>'  \
+	  -p NODETOOL_CMD_ARGS='<subcommand_arguments>'
+	```
+	- For example:
+	```
+	dcos datastax-dse plan start nodetool-ser \
+	  -p NODETOOL_CONNECTION_OPTS='-p 7199'  \
+	  -p NODETOOL_SUBCOMMAND='upgradesstables'  \
+	  -p NODETOOL_CMD_ARGS='-a'
+	```
+- Upgrading the SDK from the 0.4x series to 0.5x comes with significant framework and scheduler improvements such as:
+	- adding [multi-service support](https://github.com/mesosphere/dcos-commons/releases/tag/0.50.0) and region-/zone-awareness
+	- [scheduler metrics](https://github.com/mesosphere/dcos-commons/releases/tag/0.52.0)
+	- improving logging, debuggability, and bug fixes
+	- Please read SDK-related release notes to learn more: https://github.com/mesosphere/dcos-commons/releases
+
+## Upgrading your cluster from DSE 5 to 6
+In order to upgrade your cluster from DSE 5 to DSE 6 you must do two things:
```

Comment:
We need to point people to DSE upgrade docs: https://docs.datastax.com/en/upgrade/doc/upgrade/datastax_enterprise/upgdDSE51to67.html, and say following:

- followup instructions in the [Upgrade restrictions and limitations](https://docs.datastax.com/en/upgrade/doc/upgrade/datastax_enterprise/upgdDSE51to67.html#Upgraderestrictionsandlimitations)
- if you're using CFS, then you need to copy your data from it
- drop CFS keyspaces using CQLSH: `cfs` and `cfs_archive`

Reply:
Also we need to add an example of command for upgrade of `datastax-ops`, so people could just copy it...